### ISSUE TITLE
Fails to redis.mset({a:123,b:345}) if options.keyPrefix was used

### ISSUE THREAD

**Issue by garkin** [luin/ioredis#217]
Fails to `redis.mset({a:123,b:345})` if `options.keyPrefix` was used.
 
Before handling `keyPrefix`[#](https://github.com/luin/ioredis/blob/master/lib/command.js#L49):
![image](https://cloud.githubusercontent.com/assets/422774/11915422/d5db3fc0-a6cf-11e5-80e7-c869c32d7545.png)

After handling `keyPrefix`:
![image](https://cloud.githubusercontent.com/assets/422774/11915423/d94a93d6-a6cf-11e5-9515-15ee69d05590.png)

**Comment by AVVS** [comment]
http://redis.io/commands/mset

redis.mset(key, val, key, val), instead you pass an object, it's coerced to string, hence it fails.

as far as I remember object to payload transform is applied on hmset only

**Comment by garkin** [comment]
https://github.com/luin/ioredis/blob/5983cef74da19833168ae529c6565e49abf93ca5/test/functional/transformer.js
```
describe('mset', function () {
      it('should support object', function (done) {
        var redis = new Redis();
        redis.mset({ a: 1, b: '2' }, function (err, result) {
          expect(result).to.eql('OK');
          redis.mget('a', 'b', function (err, result) {
            expect(result).to.eql(['1', '2']);
            done();
          });
        });
      });
});
```

**Comment by AVVS** [comment]
then I'm wrong and this is a bug for sure

**Comment by luin** [comment]
Fixed in 1.13.1. Thank you for pointing out!
